alpha/veneer/README.md

```diff
@@ -0,0 +1,17 @@
+
+## Veneers 
+File-Based Catalogs (FBC) are a major improvement to the imperative update graph approaches of previous versions, giving operator authors a [declarative, deterministic approach to defining their update graph](https://olm.operatorframework.io/docs/concepts/olm-architecture/operator-catalog/creating-an-update-graph/), but it can get complex, especially as the number of releases and dependencies scale.
+
+Enter the `veneer`.  The term is a bit of a misnomer, because it isn’t something pretending (or re-cladding) something, but it is instead a simpler transitional step to generating an FBC.
+
+The single characteristic that defines a veneer in this functional context is that when it is processed it will always yield a valid FBC. The tool that is used to process these veneers into the FBC product is not relevant. 
```

Comment:
I reduced still further, since this was a place where I was struggling to effectively communicate the basis.  I retreated to a simple list of the two principles as the least-complex form.